### PR TITLE
feat: add missing french translations and short link limit exceed

### PR DESCRIPTION
**Pull request by ReidyT**
This PR translates all short links messages to french and add short link limit exceed message.

### CODE REVIEW

File: src/langs/en/messages.json [hunk 1]
```diff
@@ -119,5 +119,6 @@
   "DELETE_SHORT_LINK": "The short link was successfully deleted.",
   "SHORT_LINK_ALREADY_EXISTS": "The short link's alias already exists.",
   "SHORT_LINK_NOT_FOUND": "The short link was not found.",
-  "INVALID_CONTEXT": "The given context is not valid."
+  "INVALID_CONTEXT": "The given context is not valid.",
+  "SHORT_LINK_LIMIT_EXCEED": "The limit of short links exceeded. You can't have more than one short link per context."
```
Comment (spaenleh) [inline src/langs/en/messages.json:123]: Not sure If we should use Platform or Context ? Maybe we should call it "service" ?
```suggestion
  "SHORT_LINK_LIMIT_EXCEED": "You can't have more than one short link per platform."
```

File: src/langs/fr/messages.json [hunk 2]
```diff
@@ -103,5 +103,12 @@
   "EMPTY_CURRENT_PASSWORD": "Le mot de passe actuel n'a pas été rempli. Réessayez",
   "REQUEST_MESSAGES_IMPORT_ZIP": "Le fichier ZIP est en cours de traitement. Une pause café s'impose, vous ne pensez pas ?",
   "REQUEST_MESSAGES_IMPORT_H5P": "Le dossier H5P est en cours de traitement. Patientez s'il-vous-plait.",
-  "REQUEST_MESSAGES_UPLOAD_FILES": "Le ou les fichiers sont en file d'attente pour le téléversement. Patientez s'il-vous-plait."
+  "REQUEST_MESSAGES_UPLOAD_FILES": "Le ou les fichiers sont en file d'attente pour le téléversement. Patientez s'il-vous-plait.",
+  "CREATE_SHORT_LINK": "Le lien court a été créé avec succès",
+  "EDIT_SHORT_LINK": "Le lien court a été mis à jour avec succès",
+  "DELETE_SHORT_LINK": "Le lien court a été supprimé avec succès",
+  "SHORT_LINK_ALREADY_EXISTS": "L'alias du lien court existe déjà",
+  "SHORT_LINK_NOT_FOUND": "Le lien court n'a pas été trouvé",
```
Comment (spaenleh) [inline src/langs/fr/messages.json:111]:
```suggestion
  "CREATE_SHORT_LINK": "Le lien court a été créé avec succès.",
  "EDIT_SHORT_LINK": "Le lien court a été mis à jour avec succès.",
  "DELETE_SHORT_LINK": "Le lien court a été supprimé avec succès.",
  "SHORT_LINK_ALREADY_EXISTS": "L'alias du lien court existe déjà.",
  "SHORT_LINK_NOT_FOUND": "Le lien court n'a pas été trouvé.",
```

File: src/langs/fr/messages.json [hunk 3]
```diff
@@ -103,5 +103,12 @@
...
+  "INVALID_CONTEXT": "Le contexte donné n'est pas valide.",
+  "SHORT_LINK_LIMIT_EXCEED": "La limite de liens courts a été dépassée. Vous ne pouvez pas avoir plus d'un lien court par contexte."
```
Comment (spaenleh) [inline src/langs/fr/messages.json:113]:
```suggestion
  "SHORT_LINK_LIMIT_EXCEED": "Vous ne pouvez avoir qu'un seul lien court par contexte."
```

**Review by spaenleh — CHANGES_REQUESTED** [review]

File: src/langs/en/messages.json (hunk 1 above)
Comment (ReidyT) [reply src/langs/en/messages.json:123]: Yes, I think that's a good idea because service would be clearer to users than platform or context.

**Review by spaenleh — APPROVED** [review]
Awesome ! 🎉

**Pull request merged**